app/assets/javascripts/app/discussions/index/index.html.slim

```diff
@@ -23,7 +23,7 @@ div ui-view="sidebar"
         section.CardList.CardList--withAnimation ng-show="closedDiscussions.length"
           .CardList-label Closed Discussions
           .CardList-card ng-repeat="discussion in closedDiscussions = (ctrl.discussions | filter:{open: false})"
-            discussion-card discussion="discussion"
+            discussion-card.Discussion--closed discussion="discussion"
```

Comment:
The modifier should be inside the [directive](https://github.com/subvisual/council/blob/master/app/assets/javascripts/app/components/discussion_card/templates/discussion_card.html.slim).